graphics/MoltenVK/Portfile

```diff
@@ -53,27 +43,72 @@ destroot {
         file copy ${output_dir}/MoltenVK/MoltenVK.xcframework ${destroot}${frameworks_dir}
     }
 
-    file copy ${output_dir}/macOS/bin/MoltenVKShaderConverter ${destroot}${prefix}/bin
-    file attributes ${destroot}${prefix}/bin/MoltenVKShaderConverter -permissions +x
-    file copy ${output_dir}/macOS/lib/libMoltenVK.dylib ${destroot}${prefix}/lib
+    file copy ${output_dir}/MoltenVK/dylib/macOS/libMoltenVK.dylib ${destroot}${prefix}/lib
 
     # vulkan and vk_video are provided via vulkan-headers
     file copy ${output_dir}/MoltenVK/include/MoltenVK ${destroot}${prefix}/include
 
-    if {(![variant_isset universal])} {
+    if {[variant_exists universal] && ![variant_isset universal] || ![variant_exists universal]} {
```

Comment:
@jmroot assuming you meant doing the check like this?

Reply:
That can be simplified to `![variant_isset universal] || ![variant_exists universal]`.

Reply:
@jmroot done